dev_tools/autogenerate-bloqs-notebooks-v2.py

```diff
@@ -63,11 +63,17 @@
     'chemistry/trotter/hubbard/qpe_cost_optimization.ipynb',
     'chemistry/resource_estimation.ipynb',
     'chemistry/writing_algorithms.ipynb',
-    'factoring/factoring-via-modexp.ipynb',
+    'cryptography/rsa/factoring-via-modexp.ipynb',
     'state_preparation/state_preparation_via_rotation_tutorial.ipynb',
 ]
 
 
+# --------------------------------------------------------------------------
+# -----   Root Bloqs   -----------------------------------------------------
+# --------------------------------------------------------------------------
+ROOT_BLOQS = ['cryptography/ecc/ecc.ipynb']
```

Comment:
why can't these be bloq notebooks like the rest of the bloq notebooks

Reply:
I don't understand the comment. this is an existing bloq notebook and I'm just putting a second link to it in the new section

Reply:
The idiomatic way to include bloq documentation in the TOC is by adding a `NotebookSpecV2` to the correct section `List[NotebookSpecV2]`. Each section list is added to `NB_BY_SECTION: Dict[str, List[...]]`

Reply:
yes I get that and that notebook is already there https://github.com/quantumlib/Qualtran/blob/89e916e27a9f60b59339e36bb3fa26ea2685da3f/dev_tools/qualtran_dev_tools/notebook_specs.py#L557-L565
![image](https://github.com/user-attachments/assets/bd59e495-9165-4a93-8f6c-82940c2fb033)

---

what I'm doing here is creating a new section 
![image](https://github.com/user-attachments/assets/168b18ce-9e22-4d7a-8306-5f167ed2d91a)